### ISSUE TITLE
Cypress fixture folder getting partially deleted

### ISSUE THREAD

**Issue by allout58** [nrwl/nx#1257]
## Expected Behavior

Setting the Cypress "fixturesFolder" property to the actual fixtures source folder (like `./src/fixtures`) should not delete files.

## Current Behavior

Setting the Cypress "fixturesFolder" property to the actual fixtures source folder deletes the first of my fixtures listed alphabetically.

### Steps to Reproduce

Please provide detailed steps for reproducing the issue.

1. Set the `fixturesFolder` to `'./src/fixtures'` (or whatever the relative path to the source fixture folder is)
2. Run `ng e2e` on your project
3. The test fails, saying `Error: ENOENT: no such file or directory, open `<project_root>/src/fixtures/<first_fixture_alphabetically>.json` and that file has been deleted.

### Context

Please provide any relevant information about your setup:

- @nrwl/builders: >= `7.6.0` (tried with 7.5.2, 7.6 and 7.8, where 7.5.2 worked)
- @angular/cli: 7.3.3
- `angular.json` configuration - standard
- @angular-devkit: 0.13.8
- @ngrx: 7.2.0
- @ng-bootstrap: 4.0.4

cypress.json in a project named `fil-e2e`:
```json
{
  "fileServerFolder": "../../dist/out-tsc/apps/fil-e2e",
  "integrationFolder": "../../dist/out-tsc/apps/fil-e2e/src/integration",
  "pluginsFile": "../../dist/out-tsc/apps/fil-e2e/src/plugins/index.js",
  "supportFile": false,
  "videosFolder": "../../dist/out-tsc/apps/fil-e2e/videos",
  "screenshotsFolder": "../../dist/out-tsc/apps/fil-e2e/screenshots",

  "fixturesFolder": "./src/fixtures"
}
```

## Other
I believe the issue is with the copyFixtureFolderFunction:
 https://github.com/nrwl/nx/blob/c769b5889fb35dc7acf915686f2bb68e54068345/packages/builders/src/cypress/cypress.builder.ts#L172
 I think this function should return if the configured `fixtureFolder` is the same as the source folder, possibly with some warning that this is not recommended.

I would also be willing to submit a PR for this issue if my suggested fix above sounds reasonable.

**Comment by kauppfbi** [comment]
Hi @allout58,

do you have any update on this issue?
I recently updated my dependencies and have the same behaviour now.

**Comment by allout58** [comment]
@kauppfbi I was able to work around the issue by specifying the fixtures folder similar to the other properties in `cypress.json` (so `"fixturesFolder": "../../dist/out-tsc/apps/fil-e2e/fixtures"` rather than `"fixturesFolder": "./src/fixtures"`).

**Comment by kauppfbi** [comment]
@allout58, it just worked. 
I'm wondering, why it worked before and why it was not configured like that before.

I think, the issue can be closed.

**Comment by vsavkin** [comment]
@allout58 could you provide a repro for the original issue? In 8.2, we switched to using webpack for bundling tests, so the path should be './src/fixtures'.

**Comment by allout58** [comment]
I've uploaded a basic Nx workspace I generated when we moved our primary repo to Angular 8 and Nx 8.1.2 and reproduced the issue on the `repro-nx-1257` branch. Running `ng e2e nx-workspace-eight --watch` deletes the `example.json` fixture file, then throws an error message saying that it can't find the file.

Note: The default generated `cypress.json` file has the long file path, not `./src/fixtures`.

Repro Repo: https://github.com/allout58/nx-workspace-eight/tree/repro-nx-1257
